src/evaluation/actions.py

```diff
@@ -66,6 +50,32 @@ def clear_text(action: ActionChains):
         # Perform Delete
         action.send_keys(Keys.BACKSPACE)
 
+    client = None
+
+    @staticmethod
+    def openai_call(prompt):
+
+        if not ActionUtils.client:
+            # read it from environment variable
+            key = os.environ.get('OPENAI_API_KEY')
```

Comment:
can we change this into with dotenv so we can do project specific openai key, so 

from  dotenv import load_dotenv

            load_dotenv()
            key = os.getenv('OPENAI_API_KEY')

Reply:
Done.